app/schemas/users.js

```diff
@@ -0,0 +1,41 @@
+const emailRegex = /^[a-zA-Z]+\.[a-zA-Z]+@wolox.+((co)|(ar)|(mx))$/;
+
+exports.userSchema = {
+  firstName: {
```

Comment:
Remember input body properties should be in `snake_case`. I know it seems unnecessary but in most projects it is a standard